src/main/kotlin/team/kin/forest/domain/group/adapter/output/persistence/mapper/GroupMapper.kt

```diff
@@ -0,0 +1,47 @@
+package team.kin.forest.domain.group.adapter.output.persistence.mapper
+
+import org.springframework.stereotype.Component
+import team.kin.forest.domain.group.adapter.output.persistence.entity.GroupJpaEntity
+import team.kin.forest.domain.group.domain.Group
+import team.kin.forest.domain.user.adapter.output.persistence.mapper.UserMapper
+
+@Component
+class GroupMapper (
+    private val userMapper: UserMapper
+){
+    fun toDomain(entity: GroupJpaEntity) = entity.run {
+        Group(
+            id = id,
+            name = name,
+            content = content,
+            purpose = purpose,
+            code = code,
+            groupScope = groupScope,
+            manager = userMapper.toDomain(manager)
+        )
+    }
```

Comment:
코드를 보는 입장에서 보기 편하려면 run을 쓰지 않는게 더 낫지 않을까요?? 
```suggestion
fun toDomain(entity: GroupJpaEntity) =
        Group(
            id = entity.id,
            name = entity.name,
            content = entity.content,
            purpose = entity.purpose,
            code = entity.code,
            groupScope = entity.groupScope,
            manager = userMapper.toDomain(entity.manager)
        )
```